src/timeUtils.ts

```diff
@@ -361,25 +362,33 @@ function addUnit(value: number, unit: string, isNegative: boolean) {
 }
 
 /**
- * Converts a Day of Year (DOY) string in 'YYYY-DDDDTHH:mm:ss' format into a 'YYYY-MM-DDTHH:mm:ss' formatted time string.
+ * Converts an ISO Ordinal string 'YYYY-DDDTHH:mm:ss' or DOY string 'DOYTHH:mm:ss' into a
+ * ISO 8601 (ISO UTC) time with 'Z' suffix ex. 'YYY-MM-DDTHH:mm:ssz'
  * 
- * If the input `doyString` is already in 'YYYY-MM-DDTHH:mm:ss' format, it will be returned as-is with a 'Z' suffix.
+ * If the input time is already in ISO UTC 'YYYY-MM-DDTHH:mm:ss' format, it will be returned as-is with a 'Z' suffix.
  *
- * @param {string} doyString - The DOY string to convert, e.g., "2023-154T12:34:56".
+ * @param {string} IsoOrDoyString - The ISO Ordinal string to convert, e.g., "2023-154T12:34:56".
  * @param {boolean} [includeMsecs=true] - Whether to include milliseconds in the output.
  * @returns {string | null} - A formatted date string in 'YYYY-MM-DDTHH:mm:ss[.mmm...]Z' format, or null if parsing fails.
  *
  * @example
- * convertDoyToYmd('2023-154T12:34:56') // returns '2023-05-30T12:34:56Z'
- * convertDoyToYmd('2023-154T12:34:56.789', false) // returns '2023-05-30T12:34:56Z'
- * convertDoyToYmd('2023-05-30T12:34:56') // returns '2023-05-30T12:34:56Z'
+ * convertUtcFromDoyOrIso('2023-154T12:34:56') // returns '2023-05-30T12:34:56Z'
+ * convertUtcFromDoyOrIso('154T12:34:56.789', false) // returns '2023-05-30T12:34:56Z'
+ * convertUtcFromDoyOrIso('2023-05-30T12:34:56') // returns 2023-05-30T12:34:56
```

Comment:
Should this example say that it returns the time with "Z" appended?

Reply:
oh yeah thanks for the catch